browsers/pools.mdx

```diff
@@ -0,0 +1,303 @@
+---
+title: "Browser Pools"
+description: "Pre-warm pools of browsers for immediate acquisition"
+---
+
+Browser pools let you maintain a set of pre-warmed, identically configured browsers ready for immediate use. Instead of waiting for browser creation on each request, you can acquire a browser from the pool without the startup delay and release it when done.
+
+<Info>
+Browser pools are in limited preview. [Join our Discord](https://discord.gg/FBrveQRcud) to request access or learn more.
```

Comment:
Can we link to this book demo link instead: https://calendly.com/danny-kernel/browser-poolsz

Something like "[Book a demo]..."

Reply:
https://github.com/onkernel/docs/pull/130/commits/6b8bdd67aea081631f3d6ede9e6c6f1b4ae95933